form/form_customization.rst

```diff
@@ -21,7 +21,7 @@ enough to render an entire form, including all its fields and error messages:
 .. code-block:: twig
 
     {# form is a variable passed from the controller via either
-      $this->renderForm('...', ['form' => $form])
+      $this->render('...', ['form' => $form])
       or $this->render('...', ['form' => $form->createView()]) #}
```

Comment:
This line unfortunately won't work the same https://github.com/symfony/symfony/pull/46854#discussion_r913625780. We need to be careful.

I propose to keep this PR open until I (or anyone) try to tackle this in core.

Reply:
Yes, I created this PR because currently in 6.2 the `renderForm` function is obsolete (so the documentation is acutely wrong). This branch is still under development and can therefore indeed be further modified. If you make a change to this method, please ping me to close this PR.

Reply:
Note that both methods also weren't doing the same thing before this change. So I don't see this change blocking this PR.